packages/ws/src/ws/WebSocketShard.ts

```diff
@@ -59,6 +59,17 @@ export enum WebSocketShardDestroyRecovery {
 	Resume,
 }
 
+enum ZlibErrorCodes {
+	Z_NEED_DICT = 2,
+	Z_STREAM_END = 1,
+	Z_ERRNO = -1,
+	Z_STREAM_ERROR = -2,
+	Z_DATA_ERROR = -3,
+	Z_MEM_ERROR = -4,
+	Z_BUF_ERROR = -5,
+	Z_VERSION_ERROR = -6,
+}
```

Comment:
Having these hardcoded here is not ideal.. Especially since zlib-sync exports each of these codes as constants

We could import them and construct this object using them
https://github.com/abalabahaha/zlib-sync/blob/7f1b404ba9e1bb9af1f47ca395f33e4fdfc2299c/src/zlib_sync.cc#L220-L228

(This would imply moving this object creation into the `unpackMessage` method as that's where zlib is being imported)

Reply:
Something like this perhaps

```ts
					enum ZlibErrorCodes {
						Z_NEED_DICT = zLibSync.Z_NEED_DICT,
						Z_STREAM_END = zLibSync.Z_STREAM_END,
						Z_ERRNO = zLibSync.Z_ERRNO,
						Z_STREAM_ERROR = zLibSync.Z_STREAM_ERROR,
						Z_DATA_ERROR = zLibSync.Z_DATA_ERROR,
						Z_MEM_ERROR = zLibSync.Z_MEM_ERROR,
						Z_BUF_ERROR = zLibSync.Z_BUF_ERROR,
						Z_VERSION_ERROR = zLibSync.Z_VERSION_ERROR,
					}

					error.code = ZlibErrorCodes[this.zLibSyncInflate.err];
					this.emit(WebSocketShardEvents.Error, error);
```

Reply:
Just turn the enum into a normal object please, and move it to a constants / utils place